src/Microsoft.OData.Edm/Schema/BadEntitySet.cs

```diff
@@ -56,6 +57,19 @@ public IEdmType Type
             get { return new EdmCollectionType(new EdmEntityTypeReference(new BadEntityType(String.Empty, this.Errors), false)); }
         }
 

```

Comment:
Add doc string

Reply:
The core implementations of `IEdmNavigationSource` are `EdmNavigationSource` and `CsdlSemanticsNavigationSource`

The following for implementations of `IEdmNavigationSource` are used in rare and special scenarios: `AmbiguousEntitySetBinding`, `AmbiguousSingletonBinding`, `BadEntitySet` and `UnknownEntitySet`. For these category of implementations, I have simply applied `/// <inheritdoc>`. That should suffice